Controller/AuthorizeController.php

```diff
@@ -26,8 +26,10 @@
  *
  * @author Chris Jones <leeked@gmail.com>
  */
-class AuthorizeController extends ContainerAware
+class AuthorizeController
 {
+    use ContainerAwareTrait;
```

Comment:
well, I'm quite sure that some people still run it when they run the Symfony 2.3 LTS. And it would not be good to force to run an unmaintained version of the bundle for something related to security.
And given that this bundle is low maintenance, I don't think maintaining 2 branches just to avoid 4 small lines of codes here. There is no burden here.

I think dropping support for them might be done at the same time we drop support for Symfony 2.3 (which reaches end of maintenance in 6 months)